<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1160</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qtGraphs</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>680</width>
      <height>660</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="add_ver_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="add_ver_textEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="add_ver_status_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вершин: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_ver_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>145</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добавление вершины</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_ver_info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Укажие название и вершины для связи (0 или вес ребра) через пробел</string>
    </property>
   </widget>
   <widget class="QLabel" name="dell_ver_info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>365</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Укажие название вершины для удаления</string>
    </property>
   </widget>
   <widget class="QLabel" name="del_ver_status_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>145</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="del_ver_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>145</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Удаление вершины</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_ver_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>145</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="del_ver_textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="add_ver_vername_textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="add_edge_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>185</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добавление (изменение) ребра</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_edge_f_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>От</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="add_edge_f_textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="add_edge_s_textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="add_edge_s_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>10</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>К</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="add_edge_ves_textEdit">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>210</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="add_edge_ves_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>25</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вес</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_edge_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_edge_status_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="del_edge_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>255</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Удаление ребра</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="del_edge_f_textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="del_edge_btn">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>280</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="del_edge_f_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>От</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="del_edge_s_textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="del_edge_s_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>10</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>К</string>
    </property>
   </widget>
   <widget class="QLabel" name="del_edge_status_label">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>280</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BFS_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>325</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Обход в ширину</string>
    </property>
   </widget>
   <widget class="QLabel" name="BFS_info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начальная вершина</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="BFS_textEdit">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>350</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="BFS_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="BFS_result_textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="BFS_status_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="DFS_textEdit">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>460</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="DFS_info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начальная вершина</string>
    </property>
   </widget>
   <widget class="QLabel" name="DFS_status_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DFS_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>435</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Обход в глубину</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="DFS_result_textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="DFS_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DA_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="DA_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>545</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Алгоритм Дейкстры</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DA_textEdit">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>570</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="DA_info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начальная вершина</string>
    </property>
   </widget>
   <widget class="QLabel" name="DA_status_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>570</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="DA_result_textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="example_graph_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>655</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пример графа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="example_graph_btn_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>655</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить граф</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
